via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -158,46 +169,40 @@ export default function VideoPlayerApp({
   };
 
   return (
-    <div className="w-full flex">
-      <div
-        className={classnames(
-          // This column will grow in width per the parent flex container and
-          // allow the contained media (video) to scale with available space.
-          // The column flex layout established here ensures this container fills
-          // the full height of the parent flex container
-          'grow flex flex-col p-3'
-        )}
-      >
-        <YouTubeVideoPlayer
-          videoId={videoId}
-          play={playing}
-          time={timestamp}
-          onPlayingChanged={setPlaying}
-          onTimeChanged={setTimestamp}
-        />
-      </div>
+    <div
+      data-testid="app-container"
+      className={classnames(
+        // Constrain height of interface to viewport height
+        'flex flex-col h-[100vh] min-h-0'
+      )}
+    >
       <div
+        data-testid="top-bar"
         className={classnames(
-          // Full-height column with a width allowing comfortable line lengths
-          'h-[100vh] w-[450px] flex flex-col',
-          'bg-grey-0 border-x',
-          // TODO: This is a stopgap measure to prevent controls from being
-          // interfered with (overlaid) by sidebar controls and toolbar
-          'mr-[30px]'
+          // Match height of sidebar's top-bar
+          'h-[40px] min-h-[40px]',
+          // Background is one shade darker than sidebar top-bar
```

Comment:
Maybe separate the layout and color classes here to make it more obvious what this comment is referring to.